exporter/otlphttpexporter/otlp.go

```diff
@@ -16,13 +16,14 @@ import (
 	"time"
 
 	"go.uber.org/zap"
-	"google.golang.org/genproto/googleapis/rpc/status"
+	spb "google.golang.org/genproto/googleapis/rpc/status"
```

Comment:
Why rename the import here?

Reply:
this follows what the OTLP HTTP Receiver does, I thought it would be good to keep it consistent :)

Reply:
For now lets undo this change since it is unrelated to the functionality we're trying to change. We can do this in a different PR if necessary.